vignettes/getting-started-with-teal.Rmd

```diff
@@ -63,27 +63,30 @@ if (interactive()) {
   }
 </style>
 
-<!-- These images are generated using the diagrams inside `inst/design`  -->
+_Hovering_ the image shows the `teal` application generated by this code.
```

Comment:
@vedhav done here.

Also see section below that I reflowed by using sub-bullets